# HG changeset patch
# User Rik <rik@octave.org>
# Date 1355516147 28800
#      Fri Dec 14 12:15:47 2012 -0800
# Node ID 49f5962fa20d1aaca4f936395daa0bc2ac330e72
# Parent  921912c921026e5d33ebef5871c7637a25f236c2
ov-java.h: is_map must return false, java object is not a struct.

* ov-java.h (is_map): Change is_map to return false.

diff --git a/libinterp/octave-value/ov-java.h b/libinterp/octave-value/ov-java.h
--- a/libinterp/octave-value/ov-java.h
+++ b/libinterp/octave-value/ov-java.h
@@ -142,17 +142,17 @@ public:
 
   std::string java_class_name (void) const { return java_type; }
 
   octave_base_value* clone (void) const { return new octave_java (*this); }
   octave_base_value* empty_clone (void) const { return new octave_java (); }
 
   bool is_defined (void) const { return true; }
 
-  bool is_map (void) const { return true; }
+  bool is_map (void) const { return false; }
 
   bool is_java (void) const { return true; }
 
   string_vector map_keys (void) const;
 
   dim_vector dims (void) const;
 
   void print (std::ostream& os, bool pr_as_read_syntax = false) const
